packages/core/src/codewhisperer/commands/basicCommands.ts

```diff
@@ -270,6 +271,13 @@ export const notifyNewCustomizationsCmd = Commands.declare(
     }
 )
 
+function focusQAfterDelay() {
+    // this command won't work without a small delay after install
+    setTimeout(() => {
```

Comment:
Use `globals.clock.setTimeout()` instead. All time related things should be done with `globals.clock`. This allows us to stub `globals.clock` in our tests and be able to control the time there.

Reply:
done